content/tutorials/manage-the-rippled-server/troubleshooting/understanding-log-messages.md

```diff
@@ -145,6 +145,19 @@ Messages such as the following are caused by a bug in `rippled` v1.1.0 and earli
 These can be safely ignored.
 
 
+## Not deleting
+
+Messages such as the following occur when [online deletion is interrupted](online-deletion.html#interrupting-online-deletion):
+
+```text
+SHAMapStore:WRN Not deleting. state: syncing. age 25s
+```
+
+The `state` indicates the [server state](rippled-server-states.html) and the `age` indicates how many seconds since the last validated ledger was closed. (A healthy age for the last validated ledger is 7 seconds or less.)
```

Comment:
The `state` indicates the [server state](rippled-server-states.html), and the `age`